builder.nix

```diff
@@ -98,6 +100,15 @@ pkgs.stdenv.mkDerivation {
           ln -s $file $(basename $file)
         done
       fi
+    '' + lib.optionalString (files != null) ''
+      # copy files to avoid making etc read-only
+      cp -r --no-preserve=all ${files} files
+    '' + lib.optionalString (lib.versionOlder release "19") ''
```

Comment:
Also: `lib.versionOlder release "19"` will return `true` if `release` is "snapshot". Not sure if that's intended.

Reply:
It's not intended.